mergify_engine/actions/assign.py

```diff
@@ -61,8 +76,29 @@ def run(self, ctxt: context.Context, rule: rules.EvaluatedRule) -> check_api.Res
                     f"GitHub error: [{e.status_code}] `{e.message}`",
                 )
 
+            return check_api.Result(
+                check_api.Conclusion.SUCCESS,
+                "Assignees added",
+                ", ".join(assignees),
+            )
         return check_api.Result(
-            check_api.Conclusion.SUCCESS,
-            "Assignees added",
-            ", ".join(self.config["users"]),
+            check_api.Conclusion.FAILURE,
```

Comment:
That should not be a failure. The only way this happens is that the users passed an empty list to `add_users`, and in that case, if we don't want to support it, we need to block it in the configuration.

I think it's ok to accept a list of empty users since we'll add `remove_users` at some point, so in that case just report a success.